capellambse/extensions/reqif/elements.py

```diff
@@ -311,13 +312,17 @@ class AbstractRequirementsAttribute(c.GenericElement):
 
     definition = c.AttrProxyAccessor(AttributeDefinition, "definition")
 
+    value: xmltools.AttributeProperty | c.AttrProxyAccessor
+
     def __repr__(self) -> str:
         return self._short_repr_()
 
     def _short_repr_(self) -> str:
+        mytype = type(self).__name__
         if self.definition is not None:
-            return f"<{type(self).__name__} {self.definition.long_name!r} ({self.uuid})>"
-        return super()._short_repr_()
+            return f"<{mytype} {self.definition.long_name!r} ({self.uuid})>"
+        default_name = re.sub(r"([A-Z])", r" \1", mytype).split()
+        return f"<{mytype} [{' '.join(default_name)}] ({self.uuid})>"
```

Comment:
We can avoid the split-rejoin by slightly altering the regexp:

```suggestion
        default_name = re.sub(r"(?<!^)([A-Z])", r" \1", mytype)
        return f"<{mytype} [{default_name}] ({self.uuid})>"
```

I `timeit`'d it, and there isn't actually a difference in run time between the two (unlike what I expected). So whether you take it or not is up to you.